lib/Updater.php

```diff
@@ -723,7 +723,11 @@ private function recursiveDelete($folder) {
 			if ($fileInfo->isDir()) {
 				$directories[] = $fileInfo->getRealPath();
 			} else {
-				$files[] = $fileInfo->getRealPath();
+                if ($fileInfo->isLink()) {
+                    $files[] = $fileInfo->getPathName();
+                } else {
+                    $files[] = $fileInfo->getRealPath();
+                }			 
```

Comment:
After the closing parenthesis in line 730 there are additional spaces. For this reason the sameCodeCheck fails. Could you please remove this whitespace, signoff the commit and push again? Its ok to create another commit. No need to create a new pr here. Thank you :+1: 
![image](https://user-images.githubusercontent.com/3902676/44671921-1eae8380-aa27-11e8-9926-dd879dd43b6c.png)

Reply:
Thanks for your info. I thought I solved this now, but unfortunatley there are still two contionuous integraten tests red... 
@MorrisJobke, @danielkesselberg Could you please help me or do this for me?
Thanks a lot ;)